README.md

```diff
@@ -215,7 +218,65 @@ OPTIONS
   --username=username  [default: anonymous] Kuzzle user
 ```
 
-_See code: [src/commands/document/get.ts](https://github.com/kuzzleio/kourou/blob/v0.6.0/src/commands/document/get.ts)_
+_See code: [src/commands/document/get.ts](https://github.com/kuzzleio/kourou/blob/v0.7.0/src/commands/document/get.ts)_
+
+## `kourou es:get INDEX ID`
+
+Get a document from ES
+
+```
+USAGE
+  $ kourou es:get INDEX ID
+
+ARGUMENTS
+  INDEX  ES Index name
+  ID     Document ID
+
+OPTIONS
+  -h, --host=host  [default: localhost] Elasticsearch server host
+  -p, --port=port  [default: 9200] Elasticsearch server port
+  --help           show CLI help
+```
+
+_See code: [src/commands/es/get.ts](https://github.com/kuzzleio/kourou/blob/v0.7.0/src/commands/es/get.ts)_
+
+## `kourou es:insert INDEX`
+
+Insert a document directly into ES (will replace if exists)
```

Comment:
```suggestion
Inserts a document directly into ES (will replace if exists)
```

Reply:
Also, why "directly"? Does this mean that there won't be real-time notifications triggered?

Reply:
Yes, because method under the `es` namespace send directly a request to Elasticsearch, it won't pass through Kuzzle